concepticondata/references/references.bib

```diff
@@ -3437,4 +3437,47 @@ @Article{Rowe1954b
   pages                               = {137-146}
 }
   
-  
+@Article {Johansson2020,
+  Author                              = {Johansson, Niklas Erben and Anikin, Andrey and Carling, Gerd and Holmer, Arthur},
+  Title                               = {The typology of sound symbolism: Defining macro-concepts via their semantic and phonetic features},
+  Journal                             = {Linguistic Typology},
+  Year                                = {2020},
+  Publisher                           = {De Gruyter Mouton},
+  Address                             = {Berlin, Boston},
+  Volume                              = {24},
+  Number                              = {2},
+  Doi                                 = {https://doi.org/10.1515/lingty-2020-2034},
+  Pages                               = {253-310}
+}
+
+@Article{Johansson2017,
+  Author                              = {Johansson, Niklas Erben},
+  Booktitle                           = {Dimensions of Iconicity},
+  Editor                              = {Bauer, Matthias and Zirker, Angelika and Fischer, Olga and Ljungberg, Christina},
+  Pages                               = {39-62},
+  Publisher                           = {John Benjamins Publishing Company},
+  Series                              = {Iconicity in Language and Literature},
+  Title                               = {Tracking linguistic primitives : The phonosemantic realization of fundamental oppositional pairs},
+  Url                                 = {http://dx.doi.org/10.1075/ill.15},
```

Comment:
leave out the URL: it is the DOI, so it is redundant, and not a real URl and a DOI in wrong from.